final_prep/annotations.rb

```diff
@@ -4,40 +4,70 @@
 
 # Build a Bear
 
+# Defines a method and the arguments it takes.
 def build_a_bear(name, age, fur, clothes, special_power)
+  # Assigns an interpolated string to the variable greeting.
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  # Assigns an array to the variable demographics.
   demographics = [name, age]
+  # Assigns an interpolated string to the variable power_saying.
   power_saying = "Did you know that I can #{special_power}?"
+  # Assigns a hash to the variable built_bear.
   built_bear = {
+    # Assigns value that is a variable to a key.
     'basic_info' => demographics,
+    # Assigns value that is a variable to a key.
```

Comment:
Excellent work going line by line through this. One tip- you may want to bring attention to whether the values here come from an argument passed into the method, or a variable declared above.